docs/docs/ProgrammingGuide/keras-support.md

```diff
@@ -0,0 +1,24 @@
+# **Keras Support**
```

Comment:
It can be moved to https://bigdl-project.github.io/master/#APIGuide/Module/

Reply:
related issue: https://github.com/intel-analytics/BigDL/issues/1987

Reply:
keep it here. That issue is not related with this PR.